include/mbedtls/ecdsa.h

```diff
@@ -52,6 +52,10 @@
  *                  this is a problem, call the function
  *                  mbedtls_ecdsa_max_sig_len instead.
  */
+#if MBEDTLS_ECP_MAX_BYTES > 124
```

Comment:
I'm sorry, I'm still not happy with that new place. The comment that explains why 124 is a few lines above, separate from this test by the documentation for the next macro. Please move this test to line 43 right after "and less than 124 [...]" that explains it.

Reply:
Oh, no need to be sorry, that was my mistake :)

Reply:
Done.